### PR TITLE
Update super-admin-import-new-content-channel-from-studio.feature

### PR DESCRIPTION
**Pull request by cpauya**
### Summary
Update the Gherkin `.feature` file re Super Admin imports new content channel from Studio to reflect new behaviour of the Import content channel feature.

### Reviewer guidance
Previous `.feature` file is [here](https://github.com/learningequality/kolibri/blob/release-v0.12.y/integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature)

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature [hunk 1]
```diff
@@ -9,46 +11,81 @@ Feature: Super admin imports content from Studio
...
+    When I check a <topic> topic checkbox
     Then I see the *Import* button is active
+      And I see the number of *resources selected* flag for the selected topic checkbox
       And I see the values for *Content selected* increase
-    When I uncheck the <topic> topic checkbox
+
+  Scenario: Uncheck a topic or subtopic
+    When I uncheck a topic checkbox
+    Then I do not see the number of *resources selected* flag for the unchecked topic
+    Given there is another topic checked
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature]: `Given` should be the first thing following the `Scenario:...`

File: integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature [hunk 2]
```diff
@@ -9,46 +11,81 @@ Feature: Super admin imports content from Studio
...
+    Then I see the *Import* button is still active
+      And I see the values for *Content selected* is deducted with the values from the unchecked topic
+    Given it is the only topic checked
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature:74]: I know we are not following the "standard" Gherkin syntax by the letter, but not sure if it's kosher to have `Givens` pop-up in the middle of a scenario... 🤔 
Maybe put them in a comment after/above the step?

**Comment by radinamatic** [comment]
Changes do not really cover the "new" behavior, they just describe the steps in more detail. Good descriptions overall @cpauya, just 2 comments!

File: integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature (hunk 2 above)
Comment (cpauya) [reply integration_testing/features/super-admin/super-admin-import-new-content-channel-from-studio.feature:74]: Thanks for the review! Will edit as per your comments. 😸

**Comment by indirectlylit** [comment]
@cpauya remember to go through the items on the contributor checklist in the PR description

**Comment by cpauya** [comment]
> @cpauya remember to go through the items on the contributor checklist in the PR description

Thanks for the feedback.  Going thru them now.

**Review by radinamatic — APPROVED** [review]
LGTM! :+1:

**Comment by indirectlylit** [comment]
thanks @cpauya  -

note for checklist items which do not apply, you can just leave them unchecked

> image

**Comment by cpauya** [comment]
Oh, thanks for the info. 👍

**Pull request merged**